### ISSUE TITLE
Background and foreground colors have a sufficient contrast ratio - axe-core Error: Cannot read property 'cells' of undefined

### ISSUE THREAD

**Issue by geekiam** [GoogleChrome/lighthouse#11274]
#### Provide the steps to reproduce
1. Run LH on https://zgraya.digital/

#### What is the current behavior?
Lighthouse throws ERROR when trying to calculate sufficient contrast ratio:

![image](https://user-images.githubusercontent.com/6225592/90397879-331b3900-e0a1-11ea-93b5-f7c8fbafc294.png)

#### What is the expected behavior?
List failing elements

#### Environment Information
* Affected Channels: DevTools
* Lighthouse version: Lighthouse 6.0.0
* Chrome version: Version 84.0.4147.125 (Official Build) (64-bit)
* Node.js version: local version - Node v14.7.0
* Operating System: macOS Catalina 10.15.6 (19G2021)

**Related issues**

**Comment by patrickhulce** [comment]
Thanks for filing @geekiam! This was fixed in 6.1.0 and should hit Chrome stable in the coming weeks (40cd7a3906) You should be able to use Chrome Canary in the meantime.
